test/helpers/fixture_flows_helper.rb

```diff
@@ -1,4 +1,16 @@
 module FixtureFlowsHelper
+  BUTTON_TEXT_AND_LINK_FIXTURE = {
```

Comment:
`FixtureFlowsHelper` is included in most of the functional tests, so I don't think it is the right place to place `BUTTON_TEXT_AND_LINK_FIXTURE` and `button_text_and_link`

The constant and the method are just expectations you want to meet, it is nothing related to fixtures; I would place this code close to the test where it is being used.